neo/Ledger/StorageItem.cs

```diff
@@ -6,35 +6,30 @@ namespace Neo.Ledger
     public class StorageItem : ICloneable<StorageItem>, ISerializable
     {
         public byte[] Value;
-        public bool IsConstant;
```

Comment:
It is designed to be expandable.

Reply:
I understand that was more expandable, but i think that in this case is better because now is isolated and we save space